### PR TITLE
UVify core updated.

### PR DESCRIPTION
**Pull request by limhyon**
**Describe problem solved by this pull request**
UVify Core update. 
- SPI4 is not routed on the board so we removed.
- Upload device name is more generalized.
- TFmini minimum value updated.

**Test data / coverage**
Fully tested with actual flight.

### CODE REVIEW

File: Tools/docker_run.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ if [ -z ${PX4_DOCKER_REPO+x} ]; then
 	echo "guessing PX4_DOCKER_REPO based on input";
 	if [[ $@ =~ .*px4_fmu.* ]]; then
 		# nuttx-px4fmu-v{1,2,3,4,5}
-		PX4_DOCKER_REPO="px4io/px4-dev-nuttx:2019-11-25"
+		PX4_DOCKER_REPO="px4io/px4-dev-nuttx:latest"
```
Comment (Jaeyoung-Lim) [inline Tools/docker_run.sh]: I think this is not a good idea since the behavior will not be consistent whenever there is a image released

Comment (dagar) [reply Tools/docker_run.sh]: Yes we definitely want the specific tag specified. This really helps when you jump between different branches where the toolchain may have changed.

File: src/drivers/distance_sensor/tfmini/TFMINI.cpp [hunk 2]
```diff
@@ -245,7 +245,7 @@ TFMINI::Run()
 	// perform collection
 	if (collect() == -EAGAIN) {
 		// reschedule to grab the missing bits, time to transmit 9 bytes @ 115200 bps
-		ScheduleClear();
+		//ScheduleClear();
```
Comment (LorenzMeier) [inline src/drivers/distance_sensor/tfmini/TFMINI.cpp]: This affects all users of the sensor - could you elaborate on this?

Comment (limhyon) [reply src/drivers/distance_sensor/tfmini/TFMINI.cpp]: https://github.com/PX4/Firmware/pull/14423 @bkueng solved this problem actually.
I found this problem during updating our boards.
Will test and will PR again.

**Comment by limhyon** [comment]
Will rebase soon.

**Comment by LorenzMeier** [comment]
Could you please rebase on master and back out your TFMini changes? This should now run fine.

**Comment by limhyon** [comment]
Rebased.

**Comment by Jaeyoung-Lim** [comment]
@limhyon I don't think the branch is properly rebased. It looks like you merged master into your branch

**Comment by limhyon** [comment]
I rebased and raised another PR. #14481. So I closed this.

**Pull request closed without merging**